# RPY message - roll-pitch-yaw signals
Header header
float32 roll
float32 pitch
float32 yaw
float32 throttle
